src/Apps/Auctions/EndingSoonAuctions/EndingSoonAuctions.tsx

```diff
@@ -0,0 +1,70 @@
+import { ContextModule } from "@artsy/cohesion"
+import { Clickable, Message, Spacer, Text } from "@artsy/palette"
+import { EndingSoonAuctionsGridPaginationContainer } from "Apps/Auctions/Routes/EndingSoonAuctionsGrid"
+import { useAuthDialog } from "Components/AuthDialog"
+import { MetaTags } from "Components/MetaTags"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "System/SystemContext"
+import { EndingSoonAuctions_viewer$data } from "__generated__/EndingSoonAuctions_viewer.graphql"
+
+interface EndingSoonAuctionsProps {
+  viewer: EndingSoonAuctions_viewer$data
+}
+
+const EndingSoonAuctions: React.FC<EndingSoonAuctionsProps> = ({ viewer }) => {
+  const { isLoggedIn } = useSystemContext()
+  const { showAuthDialog } = useAuthDialog()
+
+  const handleClick = () => {
+    showAuthDialog({
+      mode: "Login",
+      options: {
+        title: mode => {
+          const action = mode === "Login" ? "Log in" : "Sign up"
+          return `${action} to see your personalized recommendations`
+        },
+      },
+      analytics: {
+        contextModule: ContextModule.auctionLotsEndingSoonRail,
+      },
+    })
+  }
+
+  return (
+    <>
+      <MetaTags title="Auction Lots for You Ending Soon" />
+
+      <Spacer y={4} />
+
+      <Text variant="xl">Auction Lots for You Ending Soon</Text>
+
+      <Spacer y={4} />
+
+      {!isLoggedIn && (
+        <>
+          <Message variant="warning">
+            <Clickable onClick={handleClick} textDecoration="underline">
+              Log in
+            </Clickable>{" "}
+            to see your personalized recommendations.
+          </Message>
+
+          <Spacer y={4} />
+        </>
+      )}
```

Comment:
**suggestion:** I wonder if we already have a component for this message we could use 🤔

Reply:
We don't, but this is the second place where this design is used. I guess we can create one. Will do 👍 

I'm never sure with such small components, I feel like when we create one it is often missed. Especially if the component is not in palette. Maybe it's just me 👀

Reply:
I like the idea of creating a separate component. If we already use it in two places, it makes a lot of sense.

And the component is probably easier to read.

Reply:
Done!